packages/firebase_auth/firebase_auth_web/lib/src/interop/auth.dart

```diff
@@ -294,6 +294,15 @@ class Auth extends JsObjectWrapper<auth_interop.AuthJsImpl> {
   /// Currently signed-in [User].
   User? get currentUser => User.getInstance(jsObject.currentUser);
 
+  // Returns the current tenantId for the instance.
+  String? get tenantId {
+    return jsObject.tenantId;
+  }
+
+  set tenantId(String? tenantId) {
+    jsObject.tenantId = tenantId;
```

Comment:
Shouldn't updating the tenant ID call `auth.setTenantId` on native? Like done in `auth`

Reply:
Not for this case - this is the interop between Dart & the JS SDK